packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx

```diff
@@ -132,8 +142,8 @@ export const TrackScreenDetailsTile = ({
   const currentUserId = useSelectorWeb(getUserId)
   const dispatchWeb = useDispatchWeb()
   const playingUid = useSelector(getPlayingUid)
-  const queueTrack = useSelector(getTrack)
   const isPlaying = useSelector(getPlaying)
+  const isPlayingUid = playingUid === uid
```

Comment:
so correct me if im wrong, so this is less strict as previously, where instead of seeing it's the same track and it's part of a queue, just if its the same track?

Reply:
It should be the same because `queueTrack` before was just using the `getTrack` selector which is the playing track. We were only using that to do comparisons on the uid anyway

Reply:
ah i see, thanks for clarification